docs/vis_tutorial.md

```diff
@@ -490,7 +490,7 @@ In the terminal:
 Python script: 
 
 ```python
-# !/usr/bin/env python
```

Comment:
It's better if workflow scripts use `#!/usr/bin/env python`. This figures out where Python is in your environment, versus a hardcoded path to `/usr/bin/python`.

Reply:
I wasn't sure, so I went with the more frequently used option. But I've switched that line in all tutorials to use `#!/usr/bin/env python`